data/json/itemgroups/Agriculture_Forage_Excavation/forage.json

```
@@ -172,5 +173,11 @@
     "type": "item_group",
     "subtype": "collection",
     "entries": [ { "item": "seed_salsify_raw", "prob": 10, "charges": [ 1, 2 ] }, { "item": "salsify_raw", "count": [ 1, 3 ] } ]
+  },
+  {
+    "id": "forage_wildrice",
+    "type": "item_group",
+    "subtype": "collection",
+    "entries": [ { "item": "seed_rice", "prob": 10, "charges": [ 1, 3 ] } ]
```

Since it's inside a collection you'll want the rice itself to spawn no need for the 10% probability, also can add the related harvest byproducts for flavor like the other forage results do:
```suggestion
    "entries": [ { "item": "straw_pile", "prob": 25, "count": [ 4, 6 ] }, { "item": "seed_rice", "charges": [ 1, 3 ] } ]
```